README.md

```diff
@@ -1,45 +1,57 @@
 # SCAAML: Side Channel Attacks Assisted with Machine Learning
+
 ![SCAAML banner](https://storage.googleapis.com/scaaml-public/visuals/scaaml-banner.png)
 
-SCAAML (Side Channel Attacks Assisted with Machine Learning) is a deep learning framework dedicated to side-channel attacks.
-It is written in python and run on top of TensorFlow 2.x.
+SCAAML (Side Channel Attacks Assisted with Machine Learning) is a deep learning
+framework dedicated to side-channel attacks.  It is written in python and run on
+top of TensorFlow 2.x.
 
 [![Coverage Status](https://coveralls.io/repos/github/google/scaaml/badge.svg?branch=main)](https://coveralls.io/github/google/scaaml?branch=main)
 
 ## Available components
 
-- [`scaaml/`](https://github.com/google/scaaml/tree/master/scaaml/): The SCAAML framework code. Its used by the various tools.
-- [`scaaml_intro/`](https://github.com/google/scaaml/tree/master/scaaml_intro): *A Hacker Guide To Deep Learning Based Side Channel Attacks*.
-  Code, dataset and models used in our step by step tutorial on how to use deep-learning to perform AES side-channel attacks in practice.
+-   [`scaaml/`](https://github.com/google/scaaml/tree/master/scaaml/): The
+    SCAAML framework code. Its used by the various tools.
+
+-   [`scaaml_intro/`](https://github.com/google/scaaml/tree/master/scaaml_intro):
+    *A Hacker Guide To Deep Learning Based Side Channel Attacks*.  Code, dataset
+    and models used in our step by step tutorial on how to use deep-learning to
+    perform AES side-channel attacks in practice.
```

Comment:
Should we put a warning that the scaaml tooling evolved without this demo being maintained?

Reply:
Last time I was checking the demo worked. If we deem it helpful I can do an integration test for this (another PR).